libres/lib/enkf/enkf_plot_gendata.cpp

```diff
@@ -15,10 +15,9 @@
    See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
    for more details.
 */
-
 #include <time.h>
 #include <float.h>
-
+#include <vector>
 #include <ert/util/double_vector.h>
```

Comment:
Believe this is already imported from enkf_plot_gendata.hpp. Also applies to a few others I think.

Reply:
Bad change manage on my part, that had already been cleaned up, just not pushed :-)